### PR TITLE
chore: use primordials in 40_testing.js

### PR DESCRIPTION
**Pull request by littledivy**

### CODE REVIEW

File: runtime/js/99_main.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ const core = globalThis.Deno.core;
 const ops = core.ops;
 const internals = globalThis.__bootstrap.internals;
 const primordials = globalThis.__bootstrap.primordials;
+// Used by 40_testing.js
+internals.primordials = primordials;
```
[the remark this answers is not in the document]
Comment (littledivy) [reply runtime/js/99_main.js:11]: Had the same concern, I don't think there is a way to use it in non-snapshot code without exposing it somewhere.

[the remark this answers is not in the document]
Comment (littledivy) [reply runtime/js/99_main.js:11]: An op that stashes them would still have to be exposed somewhere to be used by testing.js

[the remark this answers is not in the document]
Comment (mmastrac) [reply runtime/js/99_main.js:11]: That's true... I guess there's no good way to do this. Maybe we can turn primordials into a module "somehow".

**Comment by bartlomieju** [comment]
This will be much easier to handle with https://github.com/denoland/deno_core/pull/366 and https://github.com/denoland/deno_core/issues/363 available.

**Comment by bartlomieju** [comment]
Waiting on https://github.com/denoland/deno_core/pull/379

**Review by bartlomieju — APPROVED** [review]

**Pull request merged**